pkgs/development/libraries/chromaprint/default.nix

```diff
@@ -1,6 +1,7 @@
-{ stdenv, fetchurl, cmake, boost, ffmpeg }:
+{ lib, stdenv, fetchurl, cmake, boost, ffmpeg, zlib }:
 
-stdenv.mkDerivation rec {
+let frameworks = (import <nixpkgs> {}).darwin.apple_sdk.frameworks;
```

Comment:
Do not import `<nixpkgs>` from within nixpkgs, add `darwin` to the function arguments on line 1.